src/charts/tree/tree.js

```diff
@@ -16,14 +16,22 @@ const defaultOptions = {
       colors: {
         'default': '#ccc',
         collapsed: 'lightsteelblue',
-        parent: 'white'
+        parent: 'white',
+      },
+      opacity: {
+        'default': 1.0,
+        empty: 1.0
```

Comment:
shouldn't opacity be lower than 1.0 for empty nodes?

Reply:
Sorry about the late reply, I didn't want to assume people would want empty nodes faded, so I kept the default option as the same.  I pushed another commit with a fix and have the empty nodes and text set at 0.75 so it is not too noticeable but enough for the feature to be noticed (I find it really helpful to keep it below 0.5 so I can see nodes of interest).  I will post more details to the pull request.